cpa/tester/policies/common/enable_hard/policy.rego

```diff
@@ -0,0 +1,9 @@
+package org
+
+policy_name["enable_hard"]
+
+# if enabled twice, the violation should only be reported once
+enable_rule["some_hard_rule"]
+enable_hard["some_hard_rule"]
```

Comment:
add another rule that is solely enabled using enable_hard to make sure it gets picked up!

Reply:
done.